pkg/server/apiextensions.go

```diff
@@ -173,10 +167,19 @@ func (c *apiBindingAwareCRDLister) ListWithContext(ctx context.Context, selector
 		return crd.Spec.Names.Plural + "." + crd.Spec.Group
 	}
 
+	selectAll := selector.Empty()
+	matchesSelector := func(crd *apiextensionsv1.CustomResourceDefinition) bool {
+		if selectAll {
```

Comment:
that's the default behaviour, no?

Reply:
Matching this logic: https://github.com/kubernetes/kubernetes/blob/e7a2ce75e5df96ba6ea51d904bf2735397b3e203/staging/src/k8s.io/client-go/tools/cache/listers.go#L35-L40

Reply:
not wrong. I just thought am empty selector matches everything anyway.